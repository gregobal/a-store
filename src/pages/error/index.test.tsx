import {render} from "@testing-library/react";
import {MemoryRouter} from "react-router-dom";
import {ErrorPage} from "./index";

describe("Error Page", () => {
    it("snapshot", () => {
        const view = render(
            <MemoryRouter>
                <ErrorPage/>
            </MemoryRouter>
        );

        expect(view).toMatchSnapshot();
    });
});
