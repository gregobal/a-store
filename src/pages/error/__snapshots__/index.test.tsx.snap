// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Page snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="generic-wrapper__component_1xe8s generic-wrapper__column_1xe8s generic-wrapper__align-center_1xe8s generic-wrapper__padding-top-xl_1xe8s generic-wrapper__padding-right-xl_1xe8s generic-wrapper__padding-bottom-xl_1xe8s generic-wrapper__padding-left-xl_1xe8s"
      >
        <h1
          class="typography__component_h1nzy typography__styrene-xlarge_qohs3 typography__bold_h1nzy typography__accent_1faea"
        >
          Ошибка
        </h1>
        <span
          class="typography__primary-medium_xcdcz typography__bold_xcdcz"
        >
          Произошла непредвиденная ошибка, пока ее исправляют, попробуйте перейти на 
          <a
            class="link__component_1r33z link__primary_1b8mz"
            href="/"
          >
            <span
              class="link__text_1r33z"
            >
              Главную страницу
            </span>
          </a>
           или вернуться позже
        </span>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="generic-wrapper__component_1xe8s generic-wrapper__column_1xe8s generic-wrapper__align-center_1xe8s generic-wrapper__padding-top-xl_1xe8s generic-wrapper__padding-right-xl_1xe8s generic-wrapper__padding-bottom-xl_1xe8s generic-wrapper__padding-left-xl_1xe8s"
    >
      <h1
        class="typography__component_h1nzy typography__styrene-xlarge_qohs3 typography__bold_h1nzy typography__accent_1faea"
      >
        Ошибка
      </h1>
      <span
        class="typography__primary-medium_xcdcz typography__bold_xcdcz"
      >
        Произошла непредвиденная ошибка, пока ее исправляют, попробуйте перейти на 
        <a
          class="link__component_1r33z link__primary_1b8mz"
          href="/"
        >
          <span
            class="link__text_1r33z"
          >
            Главную страницу
          </span>
        </a>
         или вернуться позже
      </span>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
